docs/src/main/asciidoc/property-source-config.adoc

```diff
@@ -183,25 +183,24 @@ However, if the `production` profile is active, the configuration becomes:
 If both profiles are active, the property that appears last within the `ConfigMap` overwrites any preceding values.
 
 
-To tell Spring Boot which `profile` should be enabled at bootstrap, you can pass a system property to the Java
-command. To do so, you can launch your Spring Boot application with an environment variable that you can define with the OpenShift
-`DeploymentConfig` or Kubernetes `ReplicationConfig` resource file, as follows:
+To tell Spring Boot which `profile` should be enabled at bootstrap, you can pass `SPRING_PROFILES_ACTIVE` environment variable.
+ To do so, you can launch your Spring Boot application with an environment variable that you can define it in the PodSpec at the container specification.
+ Pod resource file, as follows:
 
 ====
 [source,yaml]
 ----
 apiVersion: v1
-kind: DeploymentConfig
+kind: Pod
+metadata:
+  name: pod-name
```

Comment:
Actually I think that:

```
kind: Pod
metadata:
  name: pod-name
```

should be replaced by `kind: Deployment` since a user wouldn't be expected to configure the pod, but the Deployment